roles/cloud-bootstrap/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Provision network resources
+  include: network.yml
+  tags:
+    - network
```

Comment:
Is there a plan to add any other sections to this? If the `cloud-bootstrap` role is just meant for networking, then it would be good to stay with convention and keep all tasks in `main.yml`. If we're planning on having things like user and tenant creation here eventually, then it's fine as is.